apiary-metastore-listener/src/main/java/com/expedia/apiary/extensions/metastore/listener/ApiarySnsListener.java

```diff
@@ -171,7 +175,8 @@ private void publishInsertEvent(
       List<String> files,
       List<String> fileChecksums) {
 
-    JSONObject json = createBaseMessage(eventType, dbName, tableName);
+    JSONObject json = new JSONObject();
+    json = createBaseMessage(eventType, dbName, tableName);
```

Comment:
Then you can move this back up onto line 178 like it was before.

Reply:
I am not sure what you mean by this. Am I removing anything here ?

Reply:
Got it. I have made this change.